### PR TITLE
Feature/shared controls

### LINKED ISSUE #551: Move shared controls to shared directory
## Story
Several of the test fixtures use the same controls, and are linked to files in the install_simple example. The test fixtures should all run some smoke tests for the Forseti app, and some fixtures do not have any tests if Forseti is working (they test the infrastructure).

## Proposed Solution
It would be easier to understand which controls are shared by moving them to a /integration/shared/contols directory.

## Acceptance Criteria
- All tests pass

### PR DESCRIPTION
**Pull request by gkowalski-google**
Resolves #551

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -24,6 +24,17 @@ verifier:
   fail_fast: false
 
 suites:
+  - name: custom_service_accounts
```
Comment (gkowalski-google) [inline .kitchen.yml]: Only changes in this file is to order the test fixtures alphabetically, and rename the commands where necessary.

File: Makefile [hunk 2]
```diff
@@ -43,6 +43,59 @@ docker_test_prepare:
 		$(REGISTRY_URL)/${DOCKER_IMAGE_DEVELOPER_TOOLS}:${DOCKER_TAG_VERSION_DEVELOPER_TOOLS} \
 		/usr/local/bin/execute_with_credentials.sh prepare_environment
 
+# Run kitchen create tests within the docker container
+.PHONY: docker_test_create
+docker_test_create:
```
Comment (gkowalski-google) [inline Makefile]: To run the full suite of automated tests locally, now you can do:
```
make docker_test_prepare
make docker_test_create
make docker_test_converge
make docker_test_verify
make docker_test_destroy
```

File: test/integration/shared/controls/server.rb [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "yaml"
+
+forseti_version = "2.24.0"
```
Comment (hshin-g) [inline test/integration/shared/controls/server.rb]: Should this be 2.25?

File: test/integration/shared/controls/client.rb [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "yaml"
+
+forseti_server_vm_internal_dns = attribute("forseti-server-vm-internal-dns")
+forseti_version = "2.24.0"
```
Comment (hshin-g) [inline test/integration/shared/controls/client.rb]: Should this be 2.25?

Comment (gkowalski-google) [reply test/integration/shared/controls/client.rb]: Nice catch! Somehow I managed to not update the client/server controls that were being used by the install simple fixture.. so there were multiple controls for client & server. I'm guessing that the tests got lucky and used the version of the controls that had 2.25.0 as the required version. I've made the changes locally and re-ran the tests, which are passing. Incoming commit :)

Comment (gkowalski-google) [reply test/integration/shared/controls/client.rb]: The changes aren't easy to see in the PR, I guess GitHub isn't a fan of symlinks. You can see the controls used by the install_simple [here](https://github.com/forseti-security/terraform-google-forseti/tree/feature/shared-controls/test/integration/install_simple/controls).

**Pull request merged**